lib/pbench/cli/agent/commands/tools/list.py

```diff
@@ -13,6 +13,13 @@
 from pbench.cli.agent import CliContext, pass_cli_context
 from pbench.cli.agent.commands.tools.base import ToolCommand
 from pbench.cli.agent.options import common_options
+from pbench.agent.tool_group import BadToolGroup
+
+# FIX:
+# the set of error codes is currently
+# 0
+# 1 = Bad tool group
```

Comment:
This message should be the text emitted at line 70.

Reply:
Will do. I plan to change the tests so that they check for the "correct" output first and then make sure that the program produces that output in each case.